accessx-status/src/applet.c

```diff
@@ -1634,11 +1634,18 @@ accessx_status_applet_fill (MatePanelApplet* applet)
         was_realized = TRUE;
     }
 
-    g_object_connect (sapplet->applet,
-                      "destroy", accessx_status_applet_destroy, sapplet,
-                      "change-orient", accessx_status_applet_reorient, sapplet,
-                      "change-size", accessx_status_applet_resize, sapplet,
-                      NULL);
+
+    g_signal_connect (sapplet->applet, "destroy",
+                      G_CALLBACK (accessx_status_applet_destroy),
+                      sapplet);
+
+    g_signal_connect (sapplet->applet, "change-orient",
+                      G_CALLBACK (accessx_status_applet_reorient),
+                      sapplet);
+
+    g_signal_connect (sapplet->applet, "change-size",
+                      G_CALLBACK (accessx_status_applet_resize),
+                      sapplet);
```

Comment:
Actually this should not have anything to do with in or out of process, and is actually a regression from dc37d79d6e2797f7f613051261466b93fc01107d. [`g_object_connect()`](https://docs.gtk.org/gobject/method.Object.connect.html) actually expects a prefix to know how to connect the signal.  Here, what has been lost in dc37d79d6e2797f7f613051261466b93fc01107d is the `signal::` prefix.

Reply:
Your proposed fix worked, committed it and about to push